docs/stats/index.rst

```diff
@@ -79,6 +79,14 @@ with the Li \& Ma formula:
 
 TODO: More examples.
```

Comment:
Add one FC code example (just call one of the functions and show the output) on the top-level page?
(to make some "PR", i.e. make it easier for people to discover it / get started with it)